src/structlog/_base.py

```diff
@@ -156,18 +143,22 @@ def _process_event(self, method_name, event, event_kw):
         .. versionchanged:: 14.0.0
             Allow final processor to return a `dict`.
         """
-        event_dict = self._context.copy()
+        # We're typing it as Any, because processors can return more than an
+        # EventDict.
```

Comment:
could probably use `Mapping[T1, Any]` (T1 is probably str, but I don't know this code well enough to indicate that)

Reply:
oh I see later in the function the "processors" can convert it to `Any`

Reply:
Yeah it's not really Any, however it's not worth the pain to type this correctly. It's highly dynamic, introspective code. Absolute hell for static typing. 😅